TSSSF_CardGen.py

```diff
@@ -171,21 +171,22 @@
-    "{draw}": "You may draw a card from the Ship or Pony deck.",
-    "{goal}": "You may discard a Goal and draw a new one to replace it.",
-    "{search}": "You may search the Ship or Pony discard pile for a card of your choice and play it.",
+    "{draw}": "You may draw 1 card from the Ship or Pony deck.",
+    "{goal}": "You may discard 1 active Goal and draw 1 new Goal to replace it.",
+    "{search}": "You may search the Ship or Pony discard pile for 1 card of your choice and put it into your hand. If it’s still in your hand at the end of your turn, discard it.",
     "{copy}": "You may copy the power of any Pony card currently on the shipping grid, except for Changelings.",
     "{hermaphrodite}": "May count as either {male} or {female} for all Goals, Ships, and powers.",
     "{double pony}": "This card counts as 2 Ponies.",
-    "{love poison}": "Instead of playing this ship with a Pony card from your hand, or connecting two ponies already on the grid, take a Pony card from the shipping grid and reattach it elsewhere with this Ship. That card's power activates.",
-    "{keyword change}": "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card counts as having any one keyword of your choice, except pony names.",
+    "{love poison}": "Instead of playing this Ship with a Pony card from your hand, or connecting two Pony cards already on the grid, you may take a Pony card from the shipping grid and reattach it elsewhere with this Ship. That card’s power activates.",
+    "{keyword change}": "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card gains one keyword of your choice, except for Pony names.",
     "{gender change}": "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card becomes the opposite gender.",
-    "{race change}": "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card becomes a race of your choice. This cannot affect Changelings.",
-    "{timeline change}": "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card counts as {postapocalypse}.",
-    "{play from discard}": "You may choose to play the top card on the Pony discard pile with this Ship, rather than use a Pony card from your hand.",
+    "{race change}": "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card becomes the race of your choice. This cannot affect Changelings.",
+    "{timeline change}": "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card's timeline becomes {postapocalypse}.",
```

Comment:
That is not the previous wording. The 1.1.4 wording says "that Pony card becomes [AT]" and 1.1.5 says "that Pony card's timeline changes to [AT]". I compromised on "that Pony card's timeline becomes [AT]" because "becomes" is used with symbols, and [AT] by itself is not clear because most cards don't have a timeline symbol, whereas almost all have race and gender.